### ISSUE TITLE
audit complains about `Class Extension is not present in the package` when an empty category is present

### ISSUE THREAD

**Issue by dalehenrich** [GemTalk/Rowan#450]
Note the empty cateogory `*rowan-tools-deprecated` in the browser:
![image](https://user-images.githubusercontent.com/623951/49604006-a0288280-f941-11e8-8f73-31eb876d2245.png)
Note that the audit gives the error:
```
  -- Auditing package Rowan-Definitions
Class Extension is not present in the package   RwProjectDefinition #*rowan-definitions-deprecated
Class Extension is not present in the package   RwProjectDefinition #*rowan-definitions-deprecated
```
Here's the full log:
```
==============Auditing project Rowan
  -- Auditing package Rowan-Cypress-Core
  -- Auditing package Rowan-Tools-Core
Class Extension is not present in the package   RwPrjLoadTool #*rowan-tools-deprecated
Class Extension is not present in the package   RwPrjCreateTool #*rowan-tools-deprecated
Class Extension is not present in the package   RwPackageTool #*rowan-tools-deprecated
  -- Auditing package Rowan-Configurations
  -- Auditing package Rowan-GemStone-Loader
  -- Auditing package Rowan-Services-Extensions
  -- Auditing package GemStone-Interactions-Core
  -- Auditing package Rowan-Cypress-Definitions
  -- Auditing package Rowan-Url-3215
  -- Auditing package Rowan-Tools-Extensions
  -- Auditing package Rowan-Core-Definitions-Extensions
  -- Auditing package GemStone-Interactions-Kernel
  -- Auditing package Rowan-Cypress-Kernel
  -- Auditing package Rowan-Kernel
  -- Auditing package Rowan-GemStone-Loader-Extensions
  -- Auditing package Rowan-Definitions
Class Extension is not present in the package   RwProjectDefinition #*rowan-definitions-deprecated
Class Extension is not present in the package   RwProjectDefinition #*rowan-definitions-deprecated
  -- Auditing package Rowan-GemStone-Definitions
  -- Auditing package Rowan-GemStone-Core
  -- Auditing package Rowan-Services-Tests
  -- Auditing package Rowan-GemStone-Kernel
  -- Auditing package Rowan-GemStone-Specifications
  -- Auditing package Rowan-Url-Extensions
  -- Auditing package Rowan-Services-Core
  -- Auditing package Rowan-Tests
  -- Auditing package Rowan-Specifications
  -- Auditing package Rowan-GemStone-3215
  -- Auditing package Rowan-Core
  -- Auditing package Rowan-JadeServer
  -- Auditing package Rowan-Url-Core
  -- Auditing package Rowan-Tools-Kernel
ERROR 2318 , a UserDefinedError occurred (error 2318), reason:halt, Post load Rowan audit failed for project 'Rowan' (UserDefinedError)
```
To reproduce, run the following after doing a full Rowan install:
```smalltalk
	|  deprecationAction suite strm res projectNames |
	deprecationAction := Deprecated deprecatedAction.
		Deprecated doErrorOnDeprecated.
		projectNames := #( 'Rowan' 'STON' 'Cypress' 'Tonel' ).
		"audit before load"
		projectNames do: [:projectName |
			| audit |
			audit := Rowan projectTools audit auditForProjectNamed: projectName.
			audit isEmpty ifFalse: [ self error: 'Pre load Rowan audit failed for project ', projectName printString ] ].
		projectNames do: [:projectName |			
		false
			ifTrue: [
				"make sure test group is loaded ... include deprecated packages"
				Rowan projectTools load
					loadProjectNamed: projectName
					withGroupNames: #('tests' 'deprecated' 'jadeServer') ]
			ifFalse: [
				"make sure test group is loaded ... do NOT include deprecated packages"
				Rowan projectTools load
					loadProjectNamed: projectName
					withGroupNames: #('tests' 'jadeServer') ] ].
		System commit.
		"audit after load"
		projectNames do: [:projectName |
			| audit |
			audit := Rowan projectTools audit auditForProjectNamed: projectName.
			audit isEmpty ifFalse: [ self error: 'Post load Rowan audit failed for project ', projectName printString ] ].
```

**Comment by alexkogan** [comment]
I thoght that we ould be an error. How would we write it(extension) out in
tonnel format with no methods.

**Comment by dalehenrich** [comment]
@alexkogan, this happened when the `*xxx-deprecated` packages were unloaded by Rowan ... and Rowan does not remove the category from the class ... so I think the fix is to only complain if the `*` category is not empty ...

**Comment by dalehenrich** [comment]
Looks like `RwProjectAuditToolTest debug: #testClassExtension` test is not passing after fix for #450 integrated:
![image](https://user-images.githubusercontent.com/623951/49671263-1bf6fd80-fa1c-11e8-9f66-76f16981b7fb.png)
